SDL/Event.cu

```diff
@@ -2209,9 +2213,18 @@ __global__ void createTrackCandidatesFromInnerInnerInnerLowerModule(struct SDL::
                 }
                 else
                 {
-		    unsigned int trackCandidateIdx = innerInnerInnerLowerModuleArrayIndex * N_MAX_TRACK_CANDIDATES_PER_MODULE + trackCandidateModuleIdx;
+//		    unsigned int trackCandidateIdx = innerInnerInnerLowerModuleArrayIndex * N_MAX_TRACK_CANDIDATES_PER_MODULE + trackCandidateModuleIdx;
```

Comment:
here and in the following: why are these old lines still needed?

Reply:
I don't really trust myself to do things right and I was lazy to do git diffs, and so I had commented out the old lines. I can remove them now and push a new commit